solutions/swb-reference/integration-tests/tests/isolated/environments/start.test.ts

```diff
@@ -53,4 +60,69 @@ describe('environment start negative tests', () => {
       );
     }
   });
+
+  test('project was deleted', async () => {
+    const dataSetName = randomTextGenerator.getFakeText('integration-test-dataSet');
+
+    const { data: costCenter } = await adminSession.resources.costCenters.create({
+      name: `${dataSetName} cost center`,
+      accountId: setup.getSettings().get('defaultHostingAccountId'),
+      description: 'a test object'
+    });
+
+    const { data: createdProject } = await adminSession.resources.projects.create({
+      name: `${dataSetName} project`,
+      description: 'test description',
+      costCenterId: costCenter.id
+    });
+
+    const projectId = createdProject.id;
+
+    const dataSetBody = CreateDataSetRequestParser.parse({
+      storageName: settings.get('DataSetsBucketName'),
+      awsAccountId: settings.get('mainAccountId'),
+      path: dataSetName, // using same name to help potential troubleshooting
+      name: dataSetName,
+      region: settings.get('awsRegion'),
+      owner: getProjectAdminRole(createdProject.id),
+      ownerType: 'GROUP',
+      type: 'internal',
+      permissions: [
+        {
+          identity: getResearcherRole(createdProject.id),
+          identityType: 'GROUP',
+          accessLevel: 'read-only'
+        }
+      ]
+    });
+
+    const { data: dataSet } = await adminSession.resources.datasets.create(dataSetBody, false);
+    expect(dataSet).toMatchObject({
```

Comment:
Is this expectation related to the thing you're testing?

Reply:
No, it's from other test. I can remove that.